addons/udes_stock_picking_batch/views/stock_picking_batch.xml

```diff
@@ -35,6 +35,16 @@
         <xpath expr="//button[@name='action_done']" position="attributes">
           <attribute name="invisible">1</attribute>
         </xpath>
+
+        <!-- Save changes to picking_ids even when it has been set to readonly via state change -->
+        <xpath expr="//field[@name='picking_ids']" position="attributes">
+          <attribute name="force_save">1</attribute>
+        </xpath>
+
+        <!-- Show transfers page first -->
+        <xpath expr="//field[@name='move_line_ids']/parent::page" position="before">
+          <xpath expr="//field[@name='picking_ids']/parent::page" position="move"/>
```

Comment:
what is this magic? you can move a node to the position of the parent xpath??

Reply:
Yeah was added in a recent version I think... maybe Odoo 13? I can't remember. But it is very nice for things like this, didn't really want to have the redefine the child elements!

The fact that the pages don't have any suitable attributes to hook into does make it seem more like magic though.

Reply:
https://twitter.com/yenthe666/status/978227288037888000

Was actually added in Odoo 12. :)